kinds/apis/v1beta1/runtimeconfig_types.go

```diff
@@ -27,6 +27,9 @@ type RuntimeConfigSpec struct {
 	AdminConsole AdminConsoleSpec `json:"adminConsole,omitempty"`
 	// LocalArtifactMirrorPort holds the Local Artifact Mirror configuration.
 	LocalArtifactMirror LocalArtifactMirrorSpec `json:"localArtifactMirror,omitempty"`
+
+	// Private CAs holds a list of paths to trusted private CA certificate PEM files.
+	PrivateCAs []string `json:"privateCAs,omitempty"`
```

Comment:
this can't be a list of paths, it has to be the files themselves

Reply:
fixed. these are no longer used on join